### PR TITLE
GARCIA-1 name changes

### PR DESCRIPTION
**Pull request by cgarcia-iwa**
Description:

Writer name was added

### CODE REVIEW

**Review by catbox-dev — CHANGES_REQUESTED** [review]
Request changes

**Review by catbox-dev — APPROVED** [review]

File: Names.txt [hunk 1]
```diff
@@ -1 +1,3 @@
--- CAGD
\ No newline at end of file
+-- CAGD
```
Comment (catbox-dev) [inline Names.txt:1]: comentario 1

File: Names.txt [hunk 2]
```diff
@@ -1 +1,3 @@
...
+
+GARCIA DOMINGUEZ CARLOS E
```
Comment (catbox-dev) [inline Names.txt:3]: comentario 2

**Review by catbox-dev — CHANGES_REQUESTED** [review]
comentarios ejemplo

File: Names.txt [hunk 3]
```diff
@@ -1 +1,3 @@
--- CAGD
```
Comment (Ulux05) [inline Names.txt:1]: comentario 2

**Review by catbox-dev — APPROVED** [review]

**Pull request merged**